pumpkin-protocol/src/client/play/c_entity_sound_effect.rs

```diff
@@ -1,16 +1,30 @@
 use pumpkin_macros::client_packet;
-use serde::Serialize;
 
-use crate::{SoundCategory, VarInt};
+use crate::{ClientPacket, IDOrSoundEvent, SoundCategory, SoundEvent, VarInt};
+
+impl ClientPacket for CEntitySoundEffect {
```

Comment:
We usually always implement ClientPacket or ServerPacket at the under the new impl. You should do that as well so the Code follows the existing code style

Reply:
I was following the docs, it makes it seem to replace #[derive(serialize)] with the impl ClientPacket. Maybe a note in the documentation could be added?